src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -34,14 +41,20 @@ export interface ChatboxProps {
   displayPoll?: boolean;
 }
 
-export const _Chatbox: React.FC<ChatboxProps> = ({
+const NEXT_RENDER_SIZE = 50;
+
+const _ChatBox: React.FC<ChatboxProps> = ({
   messages,
   venue,
   sendMessage,
   sendThreadReply,
   deleteMessage,
   displayPoll: isDisplayedPoll,
 }) => {
+  console.assert(messages.length > 0);
```

Comment:
```suggestion
```

Reply:
removed in https://github.com/sparkletown/sparkle/pull/1922/commits/3660c406551764e6c60917460a79a253b51c4a67